src/Azure.Functions.Cli/Azure.Functions.Cli.csproj

```diff
@@ -83,5 +83,9 @@
     <PackageReference Include="Microsoft.Azure.Functions.PowerShellWorker" Version="0.1.52-alpha" />
     <PackageReference Include="Microsoft.Azure.WebJobs.Script.WebHost" Version="2.0.12265" />
     <PackageReference Include="Newtonsoft.Json" Version="11.0.2" />
+    <PackageReference Include="Buildalyzer.Workspaces" Version="2.0.0" />
+    <PackageReference Include="Microsoft.CodeAnalysis" Version="2.9.0" />
+    <PackageReference Include="Microsoft.CodeAnalysis.CSharp.Workspaces" Version="2.9.0" />
+    <PackageReference Include="Microsoft.CodeAnalysis.VisualBasic.Workspaces" Version="2.9.0" />
```

Comment:
@ahmelsayed Just to confirm, you're good with these new CLI dependencies?

Reply:
sorry for the delay, just saw these comments. I think the runtime already brings in the `CodeAnalysis` packages. I don't really have any particular concerns unless @fabiocav does.